Miniforge3/construct.yaml

```diff
@@ -1,9 +1,9 @@
-{% set version = os.environ.get("MINIFORGE_VERSION", "24.11.3-0") %}
-{% set conda_libmamba_solver_version = "24.9.0"%}
-# This file is parsed by the scripts to define
```

Comment:
i would rather keep this "intelligent" parsing. if possible.

I don't want to update a file in 3 locations since it makes using bots difficult.

Reply:
I am also in favor of keeping it, but I was not working last time (I guess it depends on the current working directory).